decentraland/screens/__snapshots__/MyAccountScreen.test.js.snap

```diff
@@ -34,7 +34,75 @@ exports[`MyAccountScreen renders the component with account 1`] = `
 />
 `;
 
-exports[`MyAccountScreen renders the component without account 1`] = `
+exports[`MyAccountScreen renders the component funding with ethers with null action (used when action is unknowed) 1`] = `
+<MyAccount
+  creationSteps={
+    Array [
+      Object {
+        "creationStatus": "GENERATING_ACCOUNT",
+        "name": "Account created",
+        "percentage": 0.25,
+        "status": "done",
+      },
+      Object {
+        "creationStatus": "FUNDING_WITH_ETH",
+        "name": "Funded with ETH",
+        "percentage": 0.25,
+        "status": "done",
+      },
+      Object {
+        "creationStatus": "FUNDING_WITH_MANA",
+        "name": "Funded with MANA tokens",
+        "percentage": 0.25,
+        "status": "missing",
+      },
+      Object {
+        "creationStatus": "APPROVING_MARKETPLACE",
+        "name": "Linked to marketplace",
+        "percentage": 0.25,
+        "status": "missing",
+      },
+    ]
+  }
+  progress={0.5}
+/>
+`;
+
+exports[`MyAccountScreen renders the component funding with mana AND approving marketplace 1`] = `
+<MyAccount
+  creationSteps={
+    Array [
+      Object {
+        "creationStatus": "GENERATING_ACCOUNT",
+        "name": "Account created",
+        "percentage": 0.25,
+        "status": "done",
+      },
+      Object {
+        "creationStatus": "FUNDING_WITH_ETH",
+        "name": "Funded with ETH",
+        "percentage": 0.25,
+        "status": "done",
+      },
+      Object {
+        "creationStatus": "FUNDING_WITH_MANA",
+        "name": "Funded with MANA tokens",
+        "percentage": 0.25,
+        "status": "done",
```

Comment:
Ah ok, so this isn't a bug though based on the way we've designed it. Both funding with mana and approving marketplace have been initiated, and until we store pending state, it's the best we can do